src/services/products.js

```diff
@@ -0,0 +1,78 @@
+import axios from 'axios';
+import Utils from './utils';
+
+export async function getAllProducts() {
+    const config = {
+        headers: {
+          'Access-Control-Allow-Origin': '*',
+          'Accept': 'application/json',
+          'Authorization': localStorage.getItem('serverest/userToken'),
+        },
+    };
+
+    return await axios
+    .get(`${Utils.getBaseUrl()}/produtos`, config)
+    .then((response) => response)
+    .catch((error) => {
+        console.error(error.message);
+    })
+}
+
+export async function registerProduct(state) {
```

Comment:
```suggestion
export async function registerProduct({ name, price, description, quantity }) {
```
Acredito que assim fica mais claro sobre o que deve receber. Ao chamar o método `registerProduct` será claro que precisa ter nome, preço, descrição e quantidade, mais próximo da realidade. Enquanto _state_ não é exatamente o que esse método precisa, mas sim o nome, preço, etc, que ele contém.

O que acha dessa sugestão?

Caso concorde será preciso ajustar esse método, o `registerUser` e `registerProductWithImage`, bem como os locais que os consomem.

Caso tenha dúvidas sobre como implementar posso fazer o ajuste aqui.

Reply:
Pode fazer a alteração? Acho que não entendi, pois talvez não sabia como fazer.

Reply:
Fiz a alteração, consegue revisar?